package.json

```diff
@@ -66,6 +66,7 @@
     "enzyme-adapter-react-16": "^1.2.0",
     "eventemitter3": "^2.0.2",
     "genbank-parser": "^1.0.0",
+    "lodash": "^4.17.15",
```

Comment:
What's this used for?

Reply:
Whoops, this is outdated, I will remove. I was using the `some` function originally but then removed it.